packages/opencensus-instrumentation-mongodb/src/mongodb.ts

```diff
@@ -20,23 +20,28 @@ import {logger} from '@opencensus/opencensus-core';
 import * as mongodb from 'mongodb';
 import * as semver from 'semver';
 import * as shimmer from 'shimmer';
-import {callbackify} from 'util';
 
+export type MongoDB = typeof mongodb;
 
-/** Http instrumentation plugin for Opencensus */
+/** MongoDB instrumentation plugin for Opencensus */
 export class MongoDBPlugin extends classes.BasePlugin {
-  readonly SERVER_FNS = ['insert', 'update', 'remove', 'auth'];
-  readonly CURSOR_FNS_FIRST = ['_find', '_getmore'];
-  readonly SPAN_MONGODB_QUERY_TYPE = 'db.mongodb.query';
-  logger: types.Logger;
+  private SERVER_FNS = ['insert', 'update', 'remove', 'auth'];
```

Comment:
`private readonly` would be more accurate.

Reply:
I agree. Changed